users/migrations/0003_populate_legal_addresses.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.1.7 on 2019-04-22 15:24
```

Comment:
Could alternatively include this in the prior migration, and have one less file.  But either way works.

```python
        migrations.CreateModel(....),
        migrations.RunPython(
            add_legal_addresses, remove_legal_addresses
        ),
```